src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -66,768 +67,819 @@ parseAuthorizeCredentials(Json::Value const& jv)
-                    uNodeIndex = beast::zero;
-                    jvResult[jss::error] = "malformedRequest";
-                }
-            }
-            else if (
-                !claim_id.isObject() ||
-                !(claim_id.isMember(sfIssuingChainDoor.getJsonName()) &&
-                  claim_id[sfIssuingChainDoor.getJsonName()].isString()) ||
-                !(claim_id.isMember(sfLockingChainDoor.getJsonName()) &&
-                  claim_id[sfLockingChainDoor.getJsonName()].isString()) ||
-                !claim_id.isMember(sfIssuingChainIssue.getJsonName()) ||
-                !claim_id.isMember(sfLockingChainIssue.getJsonName()) ||
-                !claim_id.isMember(jss::xchain_owned_claim_id))
-            {
-                jvResult[jss::error] = "malformedRequest";
-            }
-            else
-            {
-                // if not specified with a node id, a claim_id is specified by
-                // four strings defining the bridge (locking_chain_door,
-                // locking_chain_issue, issuing_chain_door, issuing_chain_issue)
-                // and the claim id sequence number.
-                auto lockingChainDoor = parseBase58<AccountID>(
-                    claim_id[sfLockingChainDoor.getJsonName()].asString());
-                auto issuingChainDoor = parseBase58<AccountID>(
-                    claim_id[sfIssuingChainDoor.getJsonName()].asString());
-                Issue lockingChainIssue, issuingChainIssue;
-                bool valid = lockingChainDoor && issuingChainDoor;
-                if (valid)
-                {
-                    try
-                    {
-                        lockingChainIssue = issueFromJson(
-                            claim_id[sfLockingChainIssue.getJsonName()]);
-                        issuingChainIssue = issueFromJson(
-                            claim_id[sfIssuingChainIssue.getJsonName()]);
-                    }
-                    catch (std::runtime_error const& ex)
-                    {
-                        valid = false;
-                        jvResult[jss::error] = "malformedRequest";
-                    }
-                }
+            jvResult[jss::error] = "malformedRequest";
+            return std::nullopt;
+        }
+    }
 
-                if (valid && claim_id[jss::xchain_owned_claim_id].isIntegral())
-                {
-                    auto seq = claim_id[jss::xchain_owned_claim_id].asUInt();
-
-                    STXChainBridge bridge_spec(
-                        *lockingChainDoor,
-                        lockingChainIssue,
-                        *issuingChainDoor,
-                        issuingChainIssue);
-                    Keylet keylet = keylet::xChainClaimID(bridge_spec, seq);
-                    uNodeIndex = keylet.key;
-                }
-            }
+    if (valid && claim_id[jss::xchain_owned_claim_id].isIntegral())
+    {
+        auto seq = claim_id[jss::xchain_owned_claim_id].asUInt();
```

Comment:
Can be `const`

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/5199/commits/3d5ef0fb972cf519f9c890126e2364e5316a014e